data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/usage_rule/AbstractUsageRule.java

```diff
@@ -46,11 +46,27 @@ public void setUsageMethod(UsageMethod usageMethod) {
         this.usageMethod = usageMethod;
     }
 
+    @Override
     public State getState() {
         return state;
     }
 
     public void setState(State state) {
         this.state = state;
     }
```

Comment:
I don't agree with the fact that an AbstractUsageRule contains a State :p 
But i guess we will look deeper into this part later (?)

Reply:
Yes I guess so :p when we'll have to use the usage rules in the algorithm I guess

Reply:
ok